src/components/home-page/_home-page.scss

```diff
@@ -0,0 +1,6 @@
+.home-page {
```

Comment:
why do you name the file with "_" at the beginning of the file name?

Reply:
As I know, when we are using "_" before filename, it tells a compiler that it should be proccessed  only where this file is imported.